neo/IO/Caching/Cache.cs

```diff
@@ -97,7 +91,7 @@ private void AddInternal(TKey key, TValue item)
             {
                 if (InnerDictionary.Count >= max_capacity)
                 {
-                    //TODO: 对PLINQ查询进行性能测试，以便确定此处使用何种算法更优（并行或串行）
+                    //TODO: Perform a performance test on the PLINQ query to determine which algorithm is better here (parallel or not)
```

Comment:
@erikzhang, how about this TODO? 
Is it still an open question? We may investigate this.

Reply:
Yes, it is.